[2020-07-22 15:25:07,362] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /mail_config_manual/ HTTP/1.1" 200 1990
[2020-07-22 15:25:14,505] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /mail_config_manual/ HTTP/1.1" 200 1990
[2020-07-22 15:25:18,552] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /mail_config_manual/ HTTP/1.1" 200 2085
[2020-07-22 15:34:13,483] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 10879
[2020-07-22 15:34:15,586] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /get_config/ HTTP/1.1" 200 6526
[2020-07-22 15:34:21,310] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /get_config/ HTTP/1.1" 200 6527
[2020-07-22 15:34:22,643] [views.py:143] [views:change_hosts] [INFO]- 请求成功！ 操作人:lazy
[2020-07-22 15:34:22,646] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /api/modify_config/ HTTP/1.1" 200 2999
[2020-07-22 15:34:24,786] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 10879
[2020-07-22 15:37:08,161] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /get_config/ HTTP/1.1" 200 6527
[2020-07-22 15:37:12,751] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /get_config/ HTTP/1.1" 200 6526
[2020-07-22 15:37:13,316] [views.py:143] [views:change_hosts] [INFO]- 请求成功！ 操作人:lazy
[2020-07-22 15:37:13,326] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /api/modify_config/ HTTP/1.1" 200 2999
[2020-07-22 15:37:18,041] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 10879
[2020-07-22 16:13:52,639] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 10879
[2020-07-22 16:13:53,744] [base.py:93] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2020-07-22 16:13:53,746] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 6727
[2020-07-22 16:13:56,098] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12790
[2020-07-22 16:13:58,081] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/echarts/ HTTP/1.1" 200 880
[2020-07-22 16:13:58,102] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/pass_stats/ HTTP/1.1" 200 4096
[2020-07-22 17:31:20,963] [base.py:93] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2020-07-22 17:31:20,966] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 6727
[2020-07-22 17:49:41,717] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:49:43,545] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/echarts/ HTTP/1.1" 200 880
[2020-07-22 17:49:43,550] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/pass_stats/ HTTP/1.1" 200 4177
[2020-07-22 17:49:53,960] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:49:55,005] [base.py:93] [base:get_response] [WARNING]- Not Found: /static/login/js/echarts.min.js
[2020-07-22 17:49:55,007] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /static/login/js/echarts.min.js HTTP/1.1" 404 104
[2020-07-22 17:49:56,035] [base.py:93] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2020-07-22 17:49:56,036] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 85
[2020-07-22 17:49:59,498] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:50:00,529] [base.py:93] [base:get_response] [WARNING]- Not Found: /static/login/js/echarts.min.js
[2020-07-22 17:50:00,531] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /static/login/js/echarts.min.js HTTP/1.1" 404 104
[2020-07-22 17:50:05,413] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:50:06,484] [base.py:93] [base:get_response] [WARNING]- Not Found: /static/login/js/echarts.min.js
[2020-07-22 17:50:06,486] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /static/login/js/echarts.min.js HTTP/1.1" 404 104
[2020-07-22 17:52:10,644] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:52:11,694] [base.py:93] [base:get_response] [WARNING]- Not Found: /static/login/js/echarts.min.js
[2020-07-22 17:52:11,696] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /static/login/js/echarts.min.js HTTP/1.1" 404 104
[2020-07-22 17:52:25,537] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:52:26,570] [base.py:93] [base:get_response] [WARNING]- Not Found: /static/login/js/echarts.min.js
[2020-07-22 17:52:26,572] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /static/login/js/echarts.min.js HTTP/1.1" 404 104
[2020-07-22 17:53:03,505] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:53:04,557] [base.py:93] [base:get_response] [WARNING]- Not Found: /static/login/js/echarts.min.js
[2020-07-22 17:53:04,559] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /static/login/js/echarts.min.js HTTP/1.1" 404 104
[2020-07-22 17:54:56,253] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:54:57,303] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/login/js/echarts.min.js HTTP/1.1" 200 383547
[2020-07-22 17:54:58,785] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/echarts/ HTTP/1.1" 200 880
[2020-07-22 17:54:58,811] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/pass_stats/ HTTP/1.1" 200 4177
[2020-07-22 17:57:13,047] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:57:14,086] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/login/js/echarts.min.js HTTP/1.1" 304 0
[2020-07-22 17:57:15,573] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/echarts/ HTTP/1.1" 200 880
[2020-07-22 17:57:15,587] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/pass_stats/ HTTP/1.1" 200 4177
[2020-07-22 17:58:03,172] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:58:04,206] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/login/js/echarts.min.js HTTP/1.1" 304 0
[2020-07-22 17:58:05,670] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/pass_stats/ HTTP/1.1" 200 4177
[2020-07-22 17:58:05,672] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/echarts/ HTTP/1.1" 200 880
[2020-07-22 17:58:27,374] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12789
[2020-07-22 17:58:28,408] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/login/js/echarts.min.js HTTP/1.1" 304 0
[2020-07-22 17:58:29,893] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/pass_stats/ HTTP/1.1" 200 4177
[2020-07-22 17:58:30,093] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/echarts/ HTTP/1.1" 200 880
[2020-07-22 17:59:02,384] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /test_report/test_report20200722110730
Traceback (most recent call last):
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\PycharmProjects\MyPro1\login\viewas\test_view\autotest_view.py", line 188, in test_report
    return render(request, 'login/reports/%s.html' % report_name, locals())
  File "D:\Programs\Python\Python37\Lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: login/reports/test_report20200722110730.html
[2020-07-22 17:59:02,396] [basehttp.py:124] [basehttp:log_message] [ERROR]- "GET /test_report/test_report20200722110730 HTTP/1.1" 500 27
[2020-07-22 17:59:13,808] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /test_report/test_report20200722110730
Traceback (most recent call last):
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\PycharmProjects\MyPro1\login\viewas\test_view\autotest_view.py", line 188, in test_report
    return render(request, 'login/reports/%s.html' % report_name, locals())
  File "D:\Programs\Python\Python37\Lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "D:\Programs\Python\Python37\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: login/reports/test_report20200722110730.html
[2020-07-22 17:59:13,819] [basehttp.py:124] [basehttp:log_message] [ERROR]- "GET /test_report/test_report20200722110730 HTTP/1.1" 500 27
[2020-07-23 09:39:53,147] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /test_report_list10/ HTTP/1.1" 200 12790
[2020-07-23 09:39:54,180] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/login/js/echarts.min.js HTTP/1.1" 304 0
[2020-07-23 09:39:55,239] [base.py:93] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2020-07-23 09:39:55,240] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 85
[2020-07-23 09:39:55,644] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/echarts/ HTTP/1.1" 200 881
[2020-07-23 09:39:55,662] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /api/pass_stats/ HTTP/1.1" 200 4370
